.github/actions/notify-imgproxy-helm.yml

```
@@ -0,0 +1,19 @@
+---
+name: Notify imgproxy/helm on updated 'configuration' part of the docs
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - docs/configuration.md
+jobs:
+  notify-imgproxy-helm:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1.1.3
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
```

Let's change the name as the current one is quite ambiguous

```suggestion
          token: ${{ secrets.HELM_REPO_ACCESS_TOKEN }}
```